src/ocm/components/clusterConfiguration/review/ReviewValidations.tsx

```diff
@@ -134,7 +134,7 @@ const ValidationsDetailCollapsed = ({ cluster }: { cluster: Cluster }) => {
     () =>
       Object.values(stringToJSON<ClusterValidationsInfo>(cluster.validationsInfo) || {})
         .flat()
-        .map((val) => val.status),
+        .map((val) => val?.status || ''),
```

Comment:
This logic cpould potentially genenrate a list of empty strings `['','','',...]`
I would append, right before the trailing comma `.filter(Boolean)` in order to remove empty strings from the resulting list.